geth/wrapper.py

```diff
@@ -89,6 +89,7 @@ def construct_test_chain_kwargs(**overrides):
     overrides.setdefault('ipc_api', ALL_APIS)
 
     overrides.setdefault('verbosity', '5')
+    overrides.setdefault('shh', True)
```

Comment:
I am not sure this should be a default.  For *most* current use cases add `shh` is going to be un-necessary overhead (which there is already a lot of).

Thoughts?